packages/deployer/test/sample-project/contracts/modules/SettingsModule.sol

```diff
@@ -0,0 +1,19 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../storage/SettingsNamespace.sol";
+import "../mixins/OwnerMixin.sol";
+
+contract SettingsModule is SettingsNamespace, OwnerMixin {
```

Comment:
Settings Module kept to show an independent Namespace that will require ownership to update